input/HSCDR1wl_0.yml

```diff
@@ -0,0 +1,20 @@
+depth_cut: 24.5
+z_edges: [0.3, 0.6]
+bin_name: bin0
+data_catalogs:
+  - ['/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_GAMA09H.fits']
+  - ['/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_GAMA15H.fits']
+  - ['/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_HECTOMAP.fits']
+  - ['/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_VVDS_part1.fits',
+     '/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_VVDS_part2.fits']
+  - ['/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_WIDE12H.fits']
+  - ['/mnt/extraspace/damonge/Datasets/HSC_DR1/HSC_WIDE_XMM.fits']
+fname_cosmos: '/mnt/extraspace/damonge/Datasets/HSC_DR1/Afterburner_reweighted_COSMOS_photoz_FDFC.fits'
+fnames_cosmos_ph:
+  - '/mnt/extraspace/damonge/Datasets/HSC_DR1/pdf-s17a_wide-9812.cat.fits'
+  - '/mnt/extraspace/damonge/Datasets/HSC_DR1/pdf-s17a_wide-9813.cat.fits'
+nbin_nz: 100
```

Comment:
same as` fname_cosmos`, only grabbed from the config inside _get_nz()

Reply:
not sure I understand